package.json

```diff
@@ -16,5 +16,6 @@
   "bugs": {
     "url": "https://github.com/SocexSolutions/agenda-v2/issues"
   },
-  "homepage": "https://github.com/SocexSolutions/agenda-v2#readme"
+  "homepage": "https://github.com/SocexSolutions/agenda-v2#readme",
+  "dependencies": {}
```

Comment:
Can you undo these changes?

Reply:
yes idek what i did to do that, just those two lines?